bleak/backends/corebluetooth/client.py

```diff
@@ -381,3 +379,28 @@ async def stop_notify(
             raise BleakError(
                 "Could not stop notify on {0}: {1}".format(characteristic.uuid, success)
             )
+
+    async def update_RSSI(self, rtn: bool = False) -> int:
```

Comment:
I do not want to store the rssi in the client. Better to have this method and use it when needed since the rssi is fluctuating all the time. Call it `get_rssi`, remove the `self.rssi` and the `rtn` argument.